### PR TITLE
Adding 1.9 update notifications (English, French and Japanese only)

### PR DESCRIPTION
**Pull request by swmitra**

### CODE REVIEW

File: updates/stable/de.json [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+            {
+                "name": "Mode toggle in Untitled document",
+                "description": "Language mode can be toggled for Untitled documents. Code coloring and Code Hints for the selected mode are also supported for Untitled documents."
+            },
+            {
+                "name": "Document toggle in split view",
+                "description": "Focus can be swapped between panes using keyboard shortcut."
+            },
+            {
+                "name": "Github organization support for Brackets extensions",
```
Comment (petetnt) [inline updates/stable/de.json]: > GitHub

File: updates/stable/de.json [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+                "description": "Github organizations can now own brackets extensions. All public owners of organization can update extensions."
```
Comment (petetnt) [inline updates/stable/de.json]: > **GitHub** organizations

> **B**rackets extensions

File: updates/stable/de.json [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
     {
+        "buildNumber": 17312,
+        "versionString": "Release 1.9",
+        "dateString": "23.03.2017",
+        "releaseNotesURL": "https://github.com/adobe/brackets/wiki/Release-Notes:-1.9",
+        "downloadURL": "http://brackets.io",
+        "newFeatures": [
+            {
+                "name": "Reverse inspect in Live Preview",
+                "description": "Brackets 1.9 ships with Reverse-inspect in Live Preview. Now clicking on an element in Live Preview highlights the corresponding tag in the source code."
```
Comment (petetnt) [inline updates/stable/de.json]: Maybe `reverse inspect`

Comment (swmitra) [inline updates/stable/de.json]: Fixed!

File: updates/stable/de.json (hunk 1 above)
Comment (swmitra) [inline updates/stable/de.json]: Fixed!

File: updates/stable/de.json (hunk 2 above)
Comment (swmitra) [inline updates/stable/de.json]: Fixed!

**Comment by petetnt** [comment]
LGTM 👍

**Pull request merged**